docs/web_advanced.rst

```diff
@@ -849,9 +849,9 @@ sources (e.g. ZeroMQ, Redis Pub/Sub, AMQP, etc.) to react to received messages
 within the application.
 
 For example the background task could listen to ZeroMQ on
-:data:`zmq.SUB` socket, process and forward retrieved messages to
+``zmq.SUB`` socket, process and forward retrieved messages to
```

Comment:
This completely removes any references effectively making that intersphinx mapping useless. Don't do this.
Instead, we need to figure out what this actually meant to reference. My bet it's "subscribe"

Reply:
Don't understand last sentence :confused: